Automattic/wp-calypso
# Tailored Flows: Remove `/setup/blog` flow

Part of https://github.com/Automattic/dotcom-forge/issues/9333

## Proposed Changes

Removes the code associated to the `/setup/blog` tailored blow since we no longer use it.

## Why are these changes being made?

To clean up unused code.

## Testing Instructions

- Go to `/setup/blog`
- Make sure it still redirects to `/start`

## Pre-merge Checklist
